src/Files.App/UserControls/Sidebar/SideBarView.xaml

```diff
@@ -59,10 +61,11 @@
 				<CompositeTransform x:Name="PaneColumnGridTransform" />
 			</Grid.RenderTransform>
 
+			<!--  Main Items  -->
 			<ItemsRepeaterScrollHost HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
 				<ScrollViewer
 					x:Name="MenuItemHostScrollViewer"
-					Padding="2,8,0,2"
+					Padding="0,8,0,0"
```

Comment:
The padding is specific, please revert this change.

Reply:
But how come?

This is annoying.
![image](https://github.com/files-community/Files/assets/62196528/33973728-0759-4a31-b84f-bf8906e4af93)

Reply:
I don't recall right now but I think it had to do with not overlapping the resize strip.

Reply:
@marcelwgn Do you remember? This file [hasn't been updated](https://github.com/files-community/Files/blame/main/src/Files.App/UserControls/SideBar/SideBarView.xaml#L65) yet ever since first SidebarView implemented.

Reply:
At this time, I'm gonna revert because this is not in the scope.

Reply:
@yaira2 your call then.

Reply:
@mdtauk fyi

Reply:
As a sidenote, design changes are out of scope for this PR.